modules/getting-started/pages/docker.adoc

```diff
@@ -134,6 +134,19 @@ $ docker exec -it $(docker ps | grep tigergraph | awk '{print $1}') /bin/sh
 $ gadmin start all
 ----
 
+. Attend to security.
++
+WARNING: Your TigerGraph image is preconfigured with a superuser having the default username and password `tigergraph`. If you do not change this, anyone with access to your database will be able to read and modify it.
++
+To xref:tigergraph-server:user-access:user-management.adoc#_change_a_users_password[change the password] of the user called `tigergraph`:
++
+[source,console]
+----
+$ gadmin ALTER PASSWORD tigergraph
+----
++
+For additional security settings, see xref:tigergraph-server:security:index.adoc[].
```

Comment:
Also add note 
Please follow best practices for securing and hardening the docker container especially when installing in a shared environment.

Reply:
I don't disagree with you. I'm trying to find a phrasing that is directive for users who SHOULD think about security, which also understanding that a high percentage of users who "try" TigerGraph with Docker are working as individuals and don't care about protecting this particular DB instance.

Reply:
The page is meant for people who install TG Software. If this is a user is performing an install that is going to be shared with additional users (over network etc.,), then they must follow "best practices". 
If this is a standalone user installing it over a private network/desktop with no access to external users, then this is not so important at the moment of 'let-me-try' but it become important when the try-environment starts getting shared with additional users.